### DISCUSSION TITLE
Unable to stake - conservative and dynamic - button dont work

### DISCUSSION THREAD

**Discussion by JackHipeek** [betfinio/public#57] [Bugs and Reports]
<img width="873" alt="Snímek obrazovky 2024-10-07 v 15 37 27" src="https://github.com/user-attachments/assets/e92814dc-bd9c-4bb3-b684-36a06197f02d">

**Comment by JackHipeek** [comment]
solve. we are under 10K limit. didnt know it. So. maybe to good make some info about it or some Notification

**Comment by bf-monster** [comment]
Hello, i see issue is resolved, we added a task to create infomation about minimum stake, betfinio/staking#30
